FHIR/GoFSH
# Update dependencies

**Description:**
The packages reported as outdated by `npm outdated` are the ones that were already listed in the dependency notes. They remain outdated due to the same reasons.

**Testing Instructions:**
Confirm that all tests pass locally. Confirm that `npm audit` reports no vulnerabilities. Confirm that `npm outdated` reports only the packages that are listed in `DEPENDENCY-NOTES.md`.

**Related Issue:**
N/A
